File: class05/java/AP0502-ParOuImpar.java
import java.util.Scanner;

public class ParOuImpar {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Digite um numero inteiro: ");
        int n = sc.nextInt();
        System.out.println(n + (n % 2 == 0 ? " e PAR" : " e IMPAR"));
    }
}

File: class05/cpp/AP0502-ParOuImpar.cpp
#include <iostream>
int main() {
    int n;
    std::cout << "Digite um numero inteiro: ";
    std::cin >> n;
    std::cout << n << (n % 2 == 0 ? " e PAR" : " e IMPAR") << std::endl;
    return 0;
}
